packages/gatsby-plugin-page-creator/src/__tests__/gatsby-node.js

```diff
@@ -1,7 +1,7 @@
 const validatePath = require(`../validate-path`)
 const createPath = require(`../create-path`)
 
-describe(`JavaScript page creator`, () => {
+describe.only(`JavaScript page creator`, () => {
```

Comment:
was this intentional to leave this here? or just for testing tests during dev?

Reply:
Oh fail, I forgot to remove after testing.  All tests ran anyway so it didn’t do anything. I will remove this.

Reply:
Ha, don't worry - that's why we have those reviews ;)